src/renderer/components/dock/create-resource.tsx

```diff
@@ -21,6 +24,22 @@ interface Props {
 @observer
 export class CreateResource extends React.Component<Props> {
   @observable error = "";
+  @observable templates:GroupSelectOption<SelectOption>[] = [];
```

Comment:
I think that we should have a store of some kind that watches for changes (using https://github.com/paulmillr/chokidar). So that if we open this component multiple times we don't reload every time.

Reply:
My guess this improvement could be moved to next PR if needed.

Reply:
I don't think we push that off, this is rather important.